Microsoft.Toolkit.Uwp.SampleApp/Controls/XamlCodeEditor.xaml.cs

```diff
@@ -121,10 +123,12 @@ public string Text
 
         public DateTime TimeSampleEditedLast { get; private set; } = DateTime.MinValue;
 
-        private CssLineStyle _errorStyle = new CssLineStyle()
+        private CssLineStyle ErrorStyle
         {
-            BackgroundColor = new SolidColorBrush(Color.FromArgb(0x00, 0xFF, 0xD6, 0xD6))
-        };
+            get => _themeListener.CurrentTheme.Equals(ApplicationTheme.Light) ? 
+                new CssLineStyle() { BackgroundColor = new SolidColorBrush(Color.FromArgb(0x00, 0xFF, 0xD6, 0xD6)) } : 
+                new CssLineStyle() { BackgroundColor = new SolidColorBrush(Color.FromArgb(0x00, 0x6E, 0x00, 0x00)) };
```

Comment:
Could we use #660000 for dark theme? That way, we would achieve a contrast of 4.5:1 to the darker blue.

Thanks @Kyaa-dost for pinging me on this PR!